launcher/container_runner.go

```diff
@@ -334,11 +353,33 @@ func (r *ContainerRunner) measureContainerClaims(ctx context.Context) error {
 		}
 	}
 
-	separator := cel.CosTlv{
-		EventType:    cel.LaunchSeparatorType,
-		EventContent: nil, // Success
+	return nil
+}
+
+// measureMemoryMonitor will measure the current memory monitoring state into the COS
+// eventlog in the AttestationAgent.
+func (r *ContainerRunner) measureMemoryMonitor(ctx context.Context) error {
+	s, err := systemctl.New()
```

Comment:
probably don't need to create a new systemctl client here, you can try to pass it in as a parameter.

Reply:
Alternatively, try to get the MemoryMonitor status outside of this function

Reply:
Done, passed systemcl as a parameter.